devel/ci/integration/tests/test_bodhi_cli.py

```diff
@@ -144,7 +144,7 @@ def test_composes_info(bodhi_container, db_container):
      Updated: {compose['state_date'].strftime("%Y-%m-%d %H:%M:%S")}
 """
     # If the compose doesn't have a error_message, the CLI does not render the Error: line.
-    if compose['error_message'] is not None:
+    if compose['error_message'] is not None and compose['error_message'] != "":
```

Comment:
I think this could be written as `if compose['error_message']:`. Both `""` and `None` are considered `False`.